@xen-orchestra/backups/_VmBackup.js

```diff
@@ -320,6 +322,8 @@ exports.VmBackup = class VmBackup {
   }
 
   async run() {
+    await asyncMap(this._writers, writer => writer.cleanCorruptedBackups?.())
```

Comment:
IMHO, it would be best to make this method more generic and a part of the main writer interface.

The default implementation would be empty for copy writers and clean  corrupted backups for export writers.

See https://github.com/vatesfr/xen-orchestra/pull/5727#discussion_r613184431

Reply:
What function will have this method in case of a replication?

Reply:
Nothing at the moment, but as it's called on all writers, it needs to be part of the `AbstractWriter` interface.

Reply:
Should I call this method `init`?

Reply:
IMHO `beforeBackup` would be clearer.